sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -13,7 +13,10 @@ import {
   TextLine as TextLineModel,
   GeneratedClientGetAnalyzeFormResultResponse as GetAnalyzeFormResultResponse,
   GeneratedClientGetAnalyzeLayoutResultResponse as GetAnalyzeLayoutResultResponse,
-  GeneratedClientGetCustomModelResponse as GetCustomModelResponse
+  GeneratedClientGetCustomModelResponse as GetCustomModelResponse,
```

Comment:
Putting this on a random line, but I don't see where the FieldValue type of "selectionMark" is handled and converted to FormField. I think it needs to be added here: https://github.com/Azure/azure-sdk-for-js/blob/e72ff3ee97f7acd8df2fe91bed9be763a1517cb3/sdk/formrecognizer/ai-form-recognizer/src/transforms.ts#L212